Tasks/DotNetCoreCLIV2/dotnetcore.ts

```diff
@@ -321,26 +322,52 @@ export class dotNetExe {
         var projectPattern = this.projects;
         var searchWebProjects = this.isPublishCommand() && this.publishWebProjects;
         if (searchWebProjects) {
-            projectPattern = ["**/*.csproj", "**/*.vbproj", "**/*.fsproj"];
```

Comment:
This would be breaking existing customers. Example: if they have vbproj with web.config file, then this task will start failing for them.

Reply:
We only need to remove .vbproj from the new way (sdk tag value) of detecting web project.

Reply:
Yeah but we are not supporting web projects with vbproj, yes?